### PR TITLE
Fix about inf loop

### PR DESCRIPTION
**Pull request by mcmcgrath13**
* avoid infinite render loop in about page
* various other cleanups to get fewer errors

### CODE REVIEW

**Comment by mcmcgrath13** [comment]
This seems to fix the hang I was seeing.  To reproduce that error go to the `/storage` route then via the navbar go to `about > opportunities`.  On prod, this hangs forever, on the preview here, it seems to work now.

File: .github/workflows/firebase-qa-pull-request.yml [hunk 1]
```diff
@@ -46,5 +42,9 @@ jobs:
           expires: 7d
         env:
           FIREBASE_CLI_PREVIEWS: hostingchannels
+      - name: Deploy to Firebase
```
Comment (mirestrepo) [inline .github/workflows/firebase-qa-pull-request.yml:45]: Just out of curiosity, why the change in order on deployment?

**Review by mirestrepo — APPROVED** [review]
Thank you for fixing this! I see it's working, but I'm unsure what fixed it? The changes to `tocData`?

File: .github/workflows/firebase-qa-pull-request.yml (hunk 1 above)
Comment (mcmcgrath13) [reply .github/workflows/firebase-qa-pull-request.yml:45]: so we don't have to wait for the deployment to happen to get a preview link

**Comment by mcmcgrath13** [comment]
> Thank you for fixing this! I see it's working, but I'm unsure what fixed it? The changes to `tocData`?

That and the people sort.  `sort` is in place, and because those were acting directly on the prop data, it was modifying the prop, triggering the loop.
